lichen_cli/src/main.rs

```diff
@@ -200,9 +201,40 @@ fn main() -> color_eyre::Result<()> {
     set_colors_enabled(true);
 
     let euid = unsafe { geteuid() };
-    if euid != 0 {
-        bail!("lichen must be run as root. Re-run with sudo")
-    }
+    ensure!(euid == 0, "lichen must be run as root. Re-run with sudo.");
+
+    let partition_detection_warning = indoc! {"
```

Comment:
Personal style note, I find the curly braces from indoc's docs rather weird and would go for `indoc!()`, equivalent to `format!()`. This is a pretty weird thing about Rust: any function-like macro can be called with `()` / `[]` / `{}`.

Reply:
Noted.